caddyconfig/caddyfile/lexer.go

```diff
@@ -35,9 +35,11 @@ type (
 
 	// Token represents a single parsable unit.
 	Token struct {
-		File string
-		Line int
-		Text string
+		File        string
+		Line        int
+		Text        string
+		InSnippet   bool
+		SnippetName string
```

Comment:
It looks like these are used only in this package, right? Let's unexport them.

Reply:
Done